section2/reflection.md

```diff
@@ -2,14 +2,28 @@
+
 2. In which ways do you currently demonstrate a Growth Mindset? In which ways do you _not_?
 
+  **I think demonstrate an understanding that mistakes are apart of the learning process but I do have a hard time in asking for help.  It's not because I am afraid to ask for help.  I think I feel like I might be missing an opportunity to learn something a certain if I dont ask for help and spend a little more time in trying to get it myself.
+
 3. In this section, we removed some of the supports that you had in Section 1. We didn't give the directions for how to run a file in node and made some directions a bit more vague, intentionally. How did that feel? What was uncomfortable about it? How did it support your learning?
 
+  **At first I did not feel very well about it.  I would get something and feel comfortable and then move onto the next thing and it would feel like I completely missed something.  Overall though it helped. I feel like it empowers me to discover the answer and figure it out for myself and that it's okay I dont know what this is yet, but I can figure it out and learn it.
+
 4. What is a conditional statement? Give one example of a daily life conditional. Give one example of where a conditional is probably used in a web application you use.
 
+  **A conditional statement is a command that tells the computer to execute certain actions, provide that certain conditions are met.  A daily life conditional would be if I had 2 cups of creamer, then I would be able to have coffee.  If I had less than 2 cups of creamer then I would not be able to have coffee. A web application that uses conditionals facebook if having the correct login information gives you access to your profile; else you dont have access if you have the wrong login information.
+
 5. How do you add multiple conditions to an `if` statement? In your own words, explain how to program reads them and determines what to do.
 
+  **You add multiple conditions to an if statement by using 'else if'.  When dealing with multiple conditions, a program will determine if the variable makes certain conditions true or false.
+
 6. What tools are available to you, as a developer, if you want to check that TWO conditions are met? Or, if you want to check that 1 of 2 conditions are met, before running a block of code?
 
-7. What questions do you still have about `if` statements and/or functions?
\ No newline at end of file
+  **I can use the Chrome Dev Tools Console to check if conditions are met before running code.
+
+7. What questions do you still have about `if` statements and/or functions?
+
+  **If i am using conditional statements do I always have to end them with 'else' ?
```

Comment:
You can omit the `else` statement. A standalone `if (condition) { doStuff }` is a pretty common pattern in JavaScript code.